app/assets/stylesheets/protocol_form.sass

```diff
@@ -46,3 +46,7 @@ textarea
 
 .gray_note 
   color: gray
+
+label.required-field:after
```

Comment:
I actually like what you did here. I wasn't aware of this functionality. There's already a class for required fields, ".required", but it didn't have this css being used on it, and instead used a function in global.js.coffee to add the *s. Could you change this class to .required and perhaps remove the javascript function and calls to it?